### ISSUE TITLE
Lights turn on after log "set homekit last boot from[DATE]"

### ISSUE THREAD

**Issue by Welli1994** [ebaauw/homebridge-hue#874]
#### Issue

Hey Folks,
I'm new to Git.
My Problem is, that sometimes Lamps in my Environment turn on "by themselves" as it seems.
I was away for one week and came home to a stagelike bright livingroom.
I could find following in my Log:
#### Log Messages

```
[25.12.2020, 02:40:00] [Hue] LED Strip Sofa: set homekit last boot from Thu Dec 24 2020 03:19:56 to Fri Dec 25 2020 02:40:00
[25.12.2020, 02:40:00] [Hue] LED Strip Sofa: set homekit on from 0 to 1
[25.12.2020, 10:40:37] [Hue] LED Strip Fernseher: set homekit last boot from n/a to Fri Dec 25 2020 10:40:37
[25.12.2020, 10:40:37] [Hue] LED Strip Fernseher: set homekit on from 0 to 1
```
 so yep, my lamps had been on since then...

#### Debug Files

[homebridge-hue.json.gz](https://github.com/ebaauw/homebridge-hue/files/5755626/homebridge-hue.json.gz)

Best Regards and a good start in a better year 2021,
Welli

**Comment by ebaauw** [comment]
The message means that the light firmware booted (powered on).  Could be because the light was temporarily disconnected from power, could be a bug in the firmware (not likely if this happens to multiple lights at the same time).  Apparently these lights turn on when powered.  Some lights support configuring the power-on behaviour, but not all.  For lights that don’t support power-on configuration, you would need to setup some automation to turn them off after they power on.  That’s where _Last Boot_ comes in handy.  Of course, if you’re (living with) one of those heretics that insist on using 20th century wall switches, you want the light to turn on when powered on.

**Comment by Welli1994** [comment]
wow thanks for that quick reply!

ok, I tried setting the deconz "PowerOn" Tag but I failed... I edited the lines as in this #1055 case but I couldn't get it to work.

**Comment by ebaauw** [comment]
Those are needed only if you want to set a power-on colour.  You need to set the _PowerOn OnOff_ attribute 0x4003 in the _On/Off_ cluster 0x0006 in the GUI.  If you read that attribute and it greys out, the light doesn't support it.

**Comment by riconr1** [comment]
In my Environment only one Ikea E14 lamp does the same. Again and again: 
Can anyone help me?

`[05/01/2021, 19:42:57] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:42:39 to Tue Jan 05 2021 19:42:57
[05/01/2021, 19:42:58] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:42:57 to Tue Jan 05 2021 19:42:58
[05/01/2021, 19:42:59] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:42:58 to Tue Jan 05 2021 19:42:59
[05/01/2021, 19:43:02] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:42:59 to Tue Jan 05 2021 19:43:02
[05/01/2021, 19:43:19] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:43:02 to Tue Jan 05 2021 19:43:19
[05/01/2021, 19:43:21] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:43:19 to Tue Jan 05 2021 19:43:21
[05/01/2021, 19:43:23] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:43:21 to Tue Jan 05 2021 19:43:23
[05/01/2021, 19:43:40] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:43:23 to Tue Jan 05 2021 19:43:40
[05/01/2021, 19:43:43] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:43:40 to Tue Jan 05 2021 19:43:43
[05/01/2021, 19:43:45] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:43:43 to Tue Jan 05 2021 19:43:45
[05/01/2021, 19:44:02] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:43:45 to Tue Jan 05 2021 19:44:02
[05/01/2021, 19:44:03] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:44:02 to Tue Jan 05 2021 19:44:03
[05/01/2021, 19:44:04] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:44:03 to Tue Jan 05 2021 19:44:04
[05/01/2021, 19:44:07] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:44:04 to Tue Jan 05 2021 19:44:07
[05/01/2021, 19:44:24] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:44:07 to Tue Jan 05 2021 19:44:24
[05/01/2021, 19:44:26] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:44:24 to Tue Jan 05 2021 19:44:26
[05/01/2021, 19:44:29] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:44:26 to Tue Jan 05 2021 19:44:29
[05/01/2021, 19:44:46] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:44:29 to Tue Jan 05 2021 19:44:46
[05/01/2021, 19:44:47] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:44:46 to Tue Jan 05 2021 19:44:47
[05/01/2021, 19:44:48] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:44:47 to Tue Jan 05 2021 19:44:48
[05/01/2021, 19:44:50] [Hue] Licht TV: set homekit last boot from Tue Jan 05 2021 19:44:48 to Tue Jan 05 2021 19:44:50`

**Comment by ebaauw** [comment]
> one Ikea E14 lamp does the same

Do you mean: turn on spontaneously?  Does the bulb remain on or does it turn off and back on?

**Comment by dalovering** [comment]
@Welli1994 @riconr1  did you find a solution to this? I'm having the same issue but my light is turning on around the same time every night in the middle of the night. Unfortunately it is my nightstand light so it wakes me up at ~2:30am :( 

I noticed that it happens shortly after homebridge UI does its scheduled update, so I've just disabled that. I've also updated the firmware on my bulb using the Philips hue bluetooth app. We'll see how it goes tonight...
```[3/21/2021, 2:32:12 AM] [Homebridge UI] Running scheduled instance backup...
[3/21/2021, 2:32:12 AM] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-mDHkAd/homebridge-backup-CB223DE2CE31.tar.gz
[3/21/2021, 2:34:45 AM] [Hue] Lamp: set homekit transition control to ""
[3/21/2021, 2:36:01 AM] [Hue] Deans nightstand: set homekit last boot from Sat Mar 20 2021 06:08:51 to Sun Mar 21 2021 02:36:01
[3/21/2021, 2:36:07 AM] [Hue] Deans nightstand: set homekit last boot from Sun Mar 21 2021 02:36:01 to Sun Mar 21 2021 02:36:07
[3/21/2021, 2:36:09 AM] [Hue] Deans nightstand: set homekit on from false to true
[3/21/2021, 2:36:13 AM] [Hue] Deans nightstand: set homekit last boot from Sun Mar 21 2021 02:36:07 to Sun Mar 21 2021 02:36:13
```

**Comment by blazejpawlak** [comment]
I'm experiencing the same. I have an Ikea lightbulb "TRÅDFRI bulb E27 opal 1000lm" controlled by ConBee 2. Each time message "set homekit last boot" appears in logs the bulb changes state for a moment (from `off` to `on` or from `on` to `off`).
@dalovering did the problem disappear for you or are you still experiencing it?

**Comment by dalovering** [comment]
@blazejpawlak it never went away 😔 I reset the bulb, updated its firmware, and no changes. I started powering it off at night from the physical switch so it wouldn’t turn on and wake me up, but when I powered it back on in the morning it would flash randomly for a few minutes. I determined it was a faulty bulb so I put in a warranty request with philips and they are sending a replacement. 

FYI the “set HomeKit last boot” comes on when a bulb is powered on or when it reboots.
